src/mqt/core/_core/operations.pyi

```diff
@@ -35,42 +35,43 @@ class Control:
 class OpType:
     __members__: ClassVar[dict[OpType, str]]  # readonly
     barrier: ClassVar[OpType]  # value = <OpType.barrier: 3>
-    classic_controlled: ClassVar[OpType]  # value = <OpType.classic_controlled: 38>
-    compound: ClassVar[OpType]  # value = <OpType.compound: 34>
-    dcx: ClassVar[OpType]  # value = <OpType.dcx: 26>
-    ecr: ClassVar[OpType]  # value = <OpType.ecr: 27>
+    classic_controlled: ClassVar[OpType]  # value = <OpType.classic_controlled: 39>
+    compound: ClassVar[OpType]  # value = <OpType.compound: 35>
+    dcx: ClassVar[OpType]  # value = <OpType.dcx: 27>
+    ecr: ClassVar[OpType]  # value = <OpType.ecr: 28>
     gphase: ClassVar[OpType]  # value = <OpType.gphase: 1>
     h: ClassVar[OpType]  # value = <OpType.h: 4>
     i: ClassVar[OpType]  # value = <OpType.i: 2>
     iswap: ClassVar[OpType]  # value = <OpType.iswap: 23>
```

Comment:
I believe you missed adding the new gate type here.

Reply:
Indeed, there was a commented line after iswap, which should have been the inverse.
Done ✅